src/components/users/Login.js

```diff
@@ -0,0 +1,88 @@
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import { loginUser } from '../../redux/actions/userActions';
+
+function Login({ handleClose, toggle, setIsLoading }) {
```

Comment:
- [ ] In the mobile view, the login form is overflowing:

![image](https://github.com/c00p75/appointment-app-frontend/assets/57421823/8eab8ed9-bb74-4b21-9b20-032ebd1e1717)

Kindly add some touch up to your styling so it does not overflow 👍

Reply:
Fixed!